imu_odometry_publisher:
    ros__parameters:
        imu_topic: footIMU/imu
        #imu_topic: waistIMU/IMU

        # Yaw Publishing Method
            # Real : IMU Yaw will be published. This will be the foot Yaw, not the Human Yaw
            # Zupt : Yaw will be updated only at Foot Zero Velocity instances
            # Stable : Yaw will be updated when the foot is approximately horizontal 
                        # This will reduce Yaw distortions when foot inclined with respect to floor
        yaw_pub_method: Stable
        # Yaw Publisher Latch
            # If true, odometry will keep publishing using last known Orientation, until an orientation update is available
            # If false, odometry is published only when an Orientation update is available
        yaw_pub_latch: true

        # Calibrate yaw angle according to initial walking direction
        calibrate_yaw: true
        # Calibration Distance
            # Calibration will be performed considering readings upto this walking distance
            # Applies to yaw calibration
        calibration_distance: 2.0
